samples/holiday-package-sample/services/samples/holidayPackage/HolidayPackage.bal

```diff
@@ -0,0 +1,265 @@
+    }
+    resource FirstResponding (http:Request request, http:Response response) {
+
+        endpoint<http:HttpClient> httpEndpoint {
+            create http:HttpClient("http://localhost:9093/", {});
+        }
+
+        map params = request.getQueryParams();
+        var departureDate, _ = (string)params.depart;
+        var returnDate, _ = (string)params.returnDate;
+        var from, _ = (string)params.from;
+        var to, _ = (string)params.to;
+        var vehicleType, _ = (string)params.vehicleType;
+        var location, _ = (string)params.location;
+
+        string flightPayload = string `{"departure":"{{departureDate}}", "returnDate":"{{returnDate}}",
+                                "from":"{{from}}", "to":"{{to}}"}`;
+        string vehiclePayload = string `{"from":"{{departureDate}}", "to":"{{returnDate}}", "vehicleType":"{{vehicleType}}"}`;
+        string hotelPayload = string `{"from":"{{departureDate}}", "to":"{{returnDate}}", "location":"{{location}}"}`;
+
+        json jsonFlightResponse;
+        json jsonVehicleResponse;
+        json jsonHotelResponse;
+        json driveSgJsonResponse;
+        json dreamCarJsonResponse;
+        json sixtJsonResponse;
+        json miramarJsonResponse;
+        json aqueenJsonResponse;
+        json elizabethJsonResponse;
+
+        fork {
+
+            worker qatarWorker {
+                var payload, err = <json>flightPayload;
+                http:Request req = {};
+                http:Response respWorkerQater = {};
+                req.setJsonPayload(payload);
+                respWorkerQater, _ = httpEndpoint.post("/detail/airline/qatarAirways", req);
+                respWorkerQater -> fork;
+            }
+
+            worker asianaWorker {
+                var payload, _ = <json>flightPayload;
+                http:Request req = {};
+                http:Response respWorkerAsiana = {};
+                req.setJsonPayload(payload);
+                respWorkerAsiana, _ = httpEndpoint.post("/detail/airline/asiana", req);
+                respWorkerAsiana -> fork;
+            }
+
+            worker emiratesWorker {
+                var payload, _ = <json>flightPayload;
+                http:Request req = {};
+                http:Response respWorkerEmirates = {};
+                req.setJsonPayload(payload);
+                respWorkerEmirates, _ = httpEndpoint.post("/detail/airline/emirates", req);
+                respWorkerEmirates -> fork;
+            }
+        }
+
+        join (some 1) (map airlineResponses) {
+
+            if (airlineResponses["qatarWorker"] != null) {
+                var resQatarWorker, err = (any[])airlineResponses["qatarWorker"];
+                var responseQatar, _ = (http:Response)(resQatarWorker[0]);
+                jsonFlightResponse = responseQatar.getJsonPayload();
+            }
+
+            if (airlineResponses["asianaWorker"] != null) {
+                var resAsianaWorker, err = (any[])airlineResponses["asianaWorker"];
```

Comment:
This `err` variable if it's not used can't we ignore it. Or we can check this `err` variable before the next line.